Resources/translations/SonataMediaBundle.de.xliff

```diff
@@ -440,7 +440,19 @@
             <trans-unit id="form.label_provider_reference">
                 <source>form.label_provider_reference</source>
                 <target>Provider Referenz</target>
-            </trans-unit>            
+            </trans-unit>
+            <trans-unit id="sonata.media.block.media">
+                <source>sonata.media.block.media</source>
+                <target>Medium</target>
```

Comment:
Medium is the correct word: http://www.merriam-webster.com/dictionary/medium. Media is the plurial of medium. Sound weird ;)